mysite/announcements/views.py

```diff
@@ -0,0 +1,45 @@
+from django.shortcuts import render
+from django.contrib.auth.models import User, Group
+from django.contrib.auth import authenticate, login
+from django.http import HttpResponseRedirect
+from django import forms
+from .forms import UserRegistrationForm
+from .models import Announcements
+
+# Create your views here.
+
+
+def home(request):
```

Comment:
No `login_required`?

Reply:
Strange I forgot it but it still works fine, I can't access the page if user is not logged in